backend/remote/remote.go

```diff
@@ -169,7 +169,8 @@ func (rf *Factory) Create(address string) (types.Backend, error) {
 		return nil, err
 	}
 
-	go r.monitorPing(rpc, dataAddress /*use replica data addr for ping tracking*/)
+	// use replica data addr for ping tracking
+	go r.monitorPing(rpc, dataAddress)
```

Comment:
Please modify on the original commit... It's easy to do `git rebase -i`, or you can use tool like `stgit` to do it.